<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Currency</class>
 <widget class="QMainWindow" name="Currency">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quart 0.1</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 5px; 
	border: 1px solid gray;
}

QPushButton:hover{
	font-size: 12px;
}

QFrame{
	border: 1px solid black;
}

QSlider::handle:horizontal{
    background-color: purple;
    height: 40px;
    width: 4px;
    margin: -15px 0px;
}</string>
   </property>
   <widget class="QSlider" name="X_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Y_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Z_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="X_lcd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Y_lcd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Z_lcd">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string>Start program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homingButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Homing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string>Load program</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px; border: 1px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px; 
border: 1px solid gray;</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>481</width>
      <height>451</height>
     </rect>
    </property>
    <widget class="QFrame" name="opengl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>451</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1031</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1031</width>
      <height>91</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #21325E;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: none;
font-family: Verdana;
font-size: 24px;
font-weight: bold;

</string>
     </property>
     <property name="text">
      <string>Universal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: none;
font-family: Verdana;
font-size: 18px;

</string>
     </property>
     <property name="text">
      <string>Q-Code Controller</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>891</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
border: 1px solid gray;</string>
     </property>
     <widget class="QComboBox" name="COM_slot">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: transparent;
</string>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="BR_slot">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px; 
border: none;
background: transparent;
</string>
      </property>
      <item>
       <property name="text">
        <string>COM1</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="connButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
padding: 0px;
margin: 0px;
</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="discButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
padding: 0px;
margin: 0px;</string>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
